InnoSetupHelper/InnoSetupLogUpdate.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Hi3Helper;
+using Hi3Helper.EncTool.Parser.InnoUninstallerLog;
+using LibISULR;
+using LibISULR.Flags;
+using LibISULR.Records;
+using System;
+using System.IO;
+using System.Linq;
+using static Hi3Helper.Logger;
+
+namespace InnoSetupHelper
+{
+    public class InnoSetupLogUpdate
+    {
+        private static readonly string[] excludeDeleteFile = new string[]
+        {
+            // Generic Files
+#if DEBUG
+            "ApplyUpdate.",
+#else
+            "ApplyUpdate.exe",
+#endif
+            "config.ini",
+            "_Temp",
+            "unins00",
+            "unins000",
+
+            // Game Files (we added this in-case the user uses the same directory as the executable)
+            "Hi3SEA", "Hi3Global", "Hi3TW", "Hi3KR", "Hi3CN", "Hi3JP", "BH3",
+            "GIGlb", "GICN", "GenshinImpact", "YuanShen", "GIBilibili",
+            "SRGlb", "SRCN", "StarRail", "HSRBilibili",
+            "ZZZGlb", "ZZZCN", "ZZZ"
```

Comment:
ZZZ will probably have Bilibili. Also, how do we handle this in the event that the user creates a custom folder name which falls outside of these ones? Maybe we would want to grab this from the config in Collapse directly?

Reply:
I believe we already have a check that warns people to not use the same folder as collapse main app does to save game files, so I think it shouldn't be necessary